src/game.h

```diff
@@ -812,6 +817,7 @@ class game
         calendar nextspawn; // The turn on which monsters will spawn next.
         calendar nextweather; // The turn on which weather will shift next.
         int next_npc_id, next_faction_id, next_mission_id; // Keep track of UIDs
+        long next_creature_id;
```

Comment:
We now have `next_creature_id` and `next_npc_id`, basically two separate id pools.  But they appear to be actually one pool, one can not have an NPC with id X *and* a monster with id X. e do we need two `next_id` variables?

----

Btw. I love the whole idea of giving all creatures ids, it's really overdue.